src/catch2/reporters/catch_reporter_automake.hpp

```diff
@@ -8,15 +8,19 @@
 #ifndef CATCH_REPORTER_AUTOMAKE_HPP_INCLUDED
 #define CATCH_REPORTER_AUTOMAKE_HPP_INCLUDED
 
+#include <catch2/interfaces/catch_interfaces_reporter.hpp>
 #include <catch2/reporters/catch_reporter_streaming_base.hpp>
+#include <catch2/internal/catch_move_and_forward.hpp>
 
 #include <string>
 
 namespace Catch {
 
     class AutomakeReporter final : public StreamingReporterBase {
     public:
-        using StreamingReporterBase::StreamingReporterBase;
+        AutomakeReporter(ReporterConfig&& _config):
+            StreamingReporterBase(CATCH_MOVE(_config))
+        {}
```

Comment:
This is really ugly, and I give it about 3 months before someone refactors it back to the original code.

Reply:
You mean to the current code, this change merely refactors it to the original code :')

Reply:
More seriously, I don't think there's many ways to work around the issue. If it was one of my projects I would probably go with the following solution:

```cpp
// GCC5 compat: work around inherited constructor issues
template<typename... Args>
AutomakeReporter(Args&&... args):
    StreamingReporterBase(std::forward<Args>(args)...)
{}
```

Compared to what I submitted, it has the advantage that you don't need to change the constructor when the parent's constructor changes, which makes the maintenance easier. On the other hand it probably makes error messages slightly harder to read.